ci-operator/jobs/openshift/release/openshift-release-master-periodics.yaml

```diff
@@ -52060,6 +52060,186 @@ periodics:
     - name: result-aggregator
       secret:
         secretName: result-aggregator
+- agent: kubernetes
+  cluster: build01
+  cron: 22 18 7,14,23,30 * *
+  decorate: true
+  decoration_config:
+    skip_cloning: true
+    timeout: 6h0m0s
+  extra_refs:
+  - base_ref: master
+    org: openshift
+    repo: release
+  labels:
+    ci-operator.openshift.io/cloud: aws
+    ci-operator.openshift.io/cloud-cluster-profile: aws-qe
+    ci-operator.openshift.io/variant: nightly-4.12
+    ci.openshift.io/generator: prowgen
+    ci.openshift.io/no-builds: "true"
+    job-release: "4.12"
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-openshift-release-master-nightly-4.12-aws-ipi-byo-route53-upgrade-rollback-oldest-supported
+  reporter_config:
+    slack:
+      channel: '#alerts-ocp-rollback-ci'
+      job_states_to_report:
+      - failure
+      - error
+      report_template: Job {{.Spec.Job}} ended with {{.Status.State}}, see <{{.Status.URL}}|here>
+  spec:
+    containers:
+    - args:
+      - --gcs-upload-secret=/secrets/gcs/service-account.json
+      - --image-import-pull-secret=/etc/pull-secret/.dockerconfigjson
+      - --lease-server-credentials-file=/etc/boskos/credentials
+      - --report-credentials-file=/etc/report/credentials
+      - --secret-dir=/secrets/ci-pull-credentials
+      - --secret-dir=/usr/local/aws-ipi-byo-route53-upgrade-rollback-oldest-supported-cluster-profile
+      - --target=aws-ipi-byo-route53-upgrade-rollback-oldest-supported
+      - --variant=nightly-4.12
```

Comment:
@shellyyang1989 we do not need to test the z strema rollback for the nightly builds.

Reply:
Please check my latest comment in the Jira https://issues.redhat.com/browse/OTA-1084